_layouts/post.html

```diff
@@ -63,37 +74,68 @@ <h3 class="post-title"><a href="{{ site.baseurl }}{{ page.url }}">{{ page.title
         </a>
       </div>
       <div class="small-1 columns">
-        <a target="_blank" 
+        <a target="_blank"
           href="https://www.linkedin.com/shareArticle?mini=true&url={{site.url}}/{{page.url}}&title={{page.title}}&source={{site.baseurl}}">
           <img src="{{ site.baseurl }}/images/icons/icon-linkedin.svg" />
         </a>
       </div>
     </div>
     </div>
-    <div class="author-box columns"> 
-      <div class="author">
-        <div class="small-2 columns">
-          <img src="{% if author[1].avatar contains 'http' %}{{ author[1].avatar }}{% else %} {{site.urlimg}}avatars/{{author.avatar}} {% endif %}" title="{{ author.name }}"/>
-        </div>
-        <div class="small-10 columns">
-          <div class="author-bio">
-            <span>
-              <strong>{{author.name}},</strong>
-            </span>
-            <span>
-              {{author.position}}
-            </span>
+    {% if authorsCount > 0 %}
+      {% for currentAuthor in page.authors %}
+        {% assign author = site.data.authors[currentAuthor] %}
+        <div class="author-box columns">
+          <div class="author">
+            <div class="small-2 columns">
+              <img src="{% if author[1].avatar contains 'http' %}{{ author[1].avatar }}{% else %} {{site.urlimg}}avatars/{{author.avatar}} {% endif %}" title="{{ author.name }}"/>
+            </div>
+            <div class="small-10 columns">
+              <div class="author-bio">
+                <span>
+                  <strong>{{author.name}},</strong>
+                </span>
+                <span>
+                  {{author.position}}
+                </span>
+              </div>
+              <div class="author-info">
+                <span>{{ author.info }}</span>
+              </div>
+              <div class="author-links">
+                <a class="author-blogs" href="{{ site.baseurl }}/blog/category/{{ currentAuthor }}">Blog Posts »</a>
+                <a class="author-github" href="https://github.com/{{ author.github }}"> GitHub </a>
+              </div>
+            </div>
           </div>
-          <div class="author-info">
-            <span>{{ author.info }}</span>
+        </div>
+      {% endfor %}
+    {% else if authorCount > 0 %}
```

Comment:
could you do something like `authors.concat(author)` and then just loop over `authors`?